src/apps/Casa/README.md

```diff
@@ -1,10 +1,10 @@
 #
 
-This example shows a TrafimageMaps application together with layers for routing
-and fare networks.
+Casa Map is a TrafimageMaps application which is offered as a commercial map application by the SBB.
```

Comment:
Casa is a sub module of `trafimage-maps` that is integrated in the SBB selling application. It is used for visualizing the planned route with different means of transportation and allows the selection of relevant fare networks.